addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -19,7 +19,7 @@ function toggleDropdown($toggles, show) {
     return Promise.all(_.map($toggles, toggle => {
         var $toggle = $(toggle);
         var $dropdown = $toggle.parent();
-        var shown = $dropdown.hasClass('show');
+        var shown = $toggle.hasClass('show');
```

Comment:
- `$dropdown` is not used anymore before the if and return.
- When you touch a line, fix the "linter" issues there are with it: `const` instead of `var` (you can also improve the lines around it, especially when it is not critical code like here)